charts/incubator/odoo/values.yaml

```diff
@@ -15,6 +15,19 @@ odoo:
 workload:
   main:
     podSpec:
+      initContainers:
+        init-db:
+          type: install
+          enabled: true
+          imageSelector: "image"
```

Comment:
afaik odoo only works with `101` uid/gid
So you might have to use in both main and init container securityContext